bundler.go

```diff
@@ -29,6 +29,9 @@ type Configuration struct {
 	// The bind configuration
 	Bind ConfigurationBind `json:"bind"`
 
+	// Build tags passed to go build
+	BuildTags string `json:"build_tags"`
```

Comment:
This is a good idea but I'd rather go even further: I'd rather add a `BuildFlags map[string]string` attribute. That way, to add your `tag` flag, you'd need to set `BuildFlags` to `map[string]string{"tag":"your value"}`